spec/forms/totp_verification_form_spec.rb

```diff
@@ -33,5 +33,23 @@
         expect(form.submit).to eq result
       end
     end
+
+    context 'when the code is not exactly Devise.otp_length digits' do
+      it 'returns FormResponse with success: false' do
+        user = build_stubbed(:user)
+        codes = %W(123abc 1234567 abcdef aaaaa\n123456\naaaaaaaaa)
```

Comment:
How is this? https://github.com/18F/identity-idp/pull/1251/commits/6cf0d7079ec0dfbd3ddd926c3bf51c213bc0fb72